src/core/StatedBeanContainer.ts

```diff
@@ -33,14 +43,44 @@ export class StatedBeanContainer extends Event {
     }
   }
 
-  getBean<T>(type: ClassType<T>, name?: string): T | undefined {
-    let bean = this.registry.getBean(type, name);
+  destroy() {
+    console.info('container destroyed.');
+  }
+
+  getBeanIdentity<T>(type: ClassType<T>, name?: string | symbol) {
+    return name || this.getBeanMetaName(type) || type.name;
```

Comment:
```suggestion
    return name || this.getBeanMetaName(type);
```

`type.name` has been covered by https://github.com/mjolnirjs/stated-bean/pull/45/files#diff-cfd00c706e313cf0dd62e45f68d4affdR57.

And also, is string based name unique enough?